app/views/users/_integrations.html.erb

```diff
@@ -4,7 +4,7 @@
   <h3>Generate a personal blog from your <%= community_name %> posts</h3>
   <div class="single-app-integration">
     <% if current_user.decorate.stackbit_integration? %>
-      <h3>Connected to Stackbit <sup>beta</sup><img src="<%= asset_path("checkmark-green.svg") %>" /></h3>
+      <h3>Connected to Stackbit <sup>beta</sup><img src="<%= width="20px" style="margin-left:2px;" asset_path("checkmark-green.svg") %>" /></h3>
```

Comment:
there's a syntax error here:

```ruby
_integrations.html.erb:7: syntax error, unexpected local variable or method, expecting ')'
web       | ...e="margin-left:2px;" asset_path("checkmark-green.svg") );@ou...
web       | ...                     ^~~~~~~~~~
web       | :
web       |   app/views/users/_integrations.html.erb:7:in `'
web       |   app/views/users/edit.html.erb:86:in `_app_views_users_edit_html_erb__2629320853194007479_222300'
```

it should be outside the `<%=`, like this:

```suggestion
      <h3>Connected to Stackbit <sup>beta</sup><img width="20px" style="margin-left:2px;" src="<%= asset_path("checkmark-green.svg") %>" /></h3>
```

Reply:
Ohh sorry about my PR name :octopus: 
And thanks for changing that !